UPGRADING.rst

```diff
@@ -0,0 +1,51 @@
+Upgrading
+---------
+
+This file documents any upgrade procedure that must be performed. Because we
+don't use a semantic version, a change that requires explicit steps to upgrade
+a is referenced by its Github issue number. After checking out a branch that
+contains a commit resolving an issue listed here, the steps listed underneath
+the issue need to be performed. When switching away from that branch, to a
+branch that does not have the listed changes, the steps would need to be
+reverted. This is all fairly informal and loosely defined. Hopefully we won't
+have too many entries in this file.
+
+#558 Deploying lambdas with Terraform
+=====================================
+
+Switching **TO** this branch
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
```

No one will switch to "this branch". It isn't even clear what "this branch" refers to. Most people will switch to a branch that includes this *change* and then they'll rebase the rest of their branches. So in essence, they'll do this once and hardly anyone will have to downgrade.

Reply:
Understood. I'll remove this heading.